### DISCUSSION TITLE
Use voice-changer for mobile or teams/whatsapp discussion ;-)

### DISCUSSION THREAD

**Discussion by Thanoy7823** [w-okada/voice-changer#878] [General]
Hello and thank you for your great work :-)
Is it possible to use voice-changer for discussion ? example: whatsapp, teams, googlemeeting ect...
I try but not knowing how to do, perhaps due to the out device selection....
Thanks !

**Comment by Eruore1997** [comment]
you would need a virtual audio software to do that
